marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx

```diff
@@ -212,26 +213,65 @@ describe("login", () => {
     loadPage.stepName(stepName).should("not.exist");
   });
 
-  it("should only enable Curate and Explorer tile for hub-central-mapping-reader", () => {
+  it("Verify hub-central-mapping-reader role privileges", () => {
     let entityTypeId = "Customer";
-    let mapStepName = "mapCustomersXML";
-    cy.loginAsTestUserWithRoles("hub-central-mapping-reader").withUI()
-      .url().should("include", "/tiles");
+    let mapStepName = "mapCustomersJSON";
+    cy.loginAsTestUserWithRoles("hub-central-mapping-reader").withRequest();
+    loginPage.postLogin();
     //All tiles but Explore and Model, should show a tooltip that says contact your administrator
     ["Load", "Model", "Run"].forEach((tile) => {
       toolbar.getToolBarIcon(tile).should("have.attr", {style: "cursor: not-allowed"});
     });
 
+    cy.log("**Navigates to Curate and opens EntityTypeId name**");
     toolbar.getCurateToolbarIcon().click();
     curatePage.toggleEntityTypeId(entityTypeId);
     curatePage.verifyTabs(entityTypeId, "be.visible", "not.exist");
+
+    cy.log("**User cannot see Matching tab**");
+    // TO DO: change this after DHFPROD-8997 is merged
+    cy.get(`#${entityTypeId} [data-rr-ui-event-key="match"]`).should("not.exist");
```

Comment:
Can we move this cy.get() to a page object file and call the method here?

Reply:
@csilva-ml done :)